master/buildbot/status/master.py

```diff
@@ -134,7 +134,8 @@ def getTitle(self):
     def getTitleURL(self):
         return self.master.config.titleURL
     def getBuildbotURL(self):
-        return self.master.config.buildbotURL
+        url = self.master.config.buildbotURL
+        return url if url.endswith('/') else url + '/'
```

Comment:
@campos-ddc could you make the fix at the point where the configuration file is read / parsed?